doc/user-guide/04-scenarios.adoc

```diff
@@ -1331,3 +1331,109 @@ Finished recovering your system. You can explore it under '/mnt/local'.
+manually.
+
+**Beware:** The `repair` workflow can only be used on the system where
+the rescue image was generated, as it bases its logic on the filesystem
+layout description file generated during the run of the `mkrescue` or
+`mkbackup` workflows.
+
+Here are the steps you would typically follow:
+
+=== Create your recovery image
+Using any of the techniques described in the other scenarios, create a 
+ReaR recovery image for your system (through `rear mkrescue` or `rear 
+mkbackup`). If you only take the `repair` workflow into consideration, it 
+doesn't matter whether you also make a backup of your system or not 
+(obviously, you'd better cover all your bases and make sure you'd be able to 
+perform a full `recover` as well should the need occur).
+
+Please note, that by default ReaR only includes in the recovery image the 
+tools it will need to recover the system. If you anticipate the need for 
+some extra tools in the context of a repair operation (e.g. tools that you 
+might need in the event `chroot`ing into the target system doesn't work), you 
+should make sure to include them in your recovery image by adding to the 
+`PROGS` or `REQUIRED_PROGS` configuration variables (please refer to the 
+comments in `default.conf` for the exact meaning of each).
+
+=== Booting on the recovery image
+Arrange for the target system to boot on your recovery image as you would in
+any of the other scenarios.
+
+=== Launching the repair workflow
+Issue the `rear repair` command to launch the workflow:
+
+----
+RESCUE pc-pan:~ # rear -v repair
+Relax-and-Recover 2.5 / Git
+Running rear repair (PID 779)
+Using log file: /var/log/rear/rear-pc-pan.log
+Running workflow repair within the ReaR rescue/recovery system
+Will do driver migration (recreating initramfs/initrd)
+Comparing disks
+Device sda has expected (same) size 34359738368 (will be used for recovery)
+Disk configuration looks identical
+UserInput -I DISK_LAYOUT_PROCEED_RECOVERY needed in /usr/share/rear/layout/prep-for-mount/default/250_compare_disks.sh line 148
+Proceed with recovery (yes) otherwise manual disk layout configuration is enforced
+(default 'yes' timeout 30 seconds)
+
+UserInput: No real user input (empty or only spaces) - using default input
+UserInput: No choices - result is 'yes'
+User confirmed to proceed with recovery
+Start target system mount.
+Mounting filesystem /
+Mounting filesystem /home
+Mounting filesystem /boot/efi
+Please enter the password for LUKS device cr_vg00-lvol4 (/dev/mapper/vg00-lvol4):
+Enter passphrase for /dev/mapper/vg00-lvol4: 
+Mounting filesystem /products
+Mounting virtual filesystems below target root.
+Target system now ready for chrooting into it.
+Disk layout processed.
+Finished recovering your system. You can explore it under '/mnt/local'.
+----
+
+As you can see in the output above, you will first be asked to confirm 
+(`Proceed with recovery`) -- simply press return. All the target filesystems 
```

Comment:
To improve the hardcoded `Proceed with recovery` you may adapt
usr/share/rear/layout/prepare/default/250_compare_disks.sh
like
```
--- usr/share/rear/layout/prepare/default/250_compare_disks.sh.original 2019-11-04 15:30:52.817824929 +0100
+++ usr/share/rear/layout/prepare/default/250_compare_disks.sh  2019-11-05 13:51:40.864378207 +0100
@@ -145 +145 @@
-    local prompt="Proceed with recovery (yes) otherwise manual disk layout configuration is enforced"
+    local prompt="Proceed with '$WORKFLOW' (yes) otherwise manual disk layout configuration is enforced"
@@ -150 +150 @@
-        is_true "$wilful_input" && LogPrint "User confirmed to proceed with recovery" || LogPrint "Proceeding with recovery by default"
+        is_true "$wilful_input" && LogPrint "User confirmed to proceed with '$WORKFLOW'" || LogPrint "Proceeding with '$WORKFLOW' by default"
```

Reply:
Changes committed.